src/features/apiDelegation/offered/ReceiptPage/ReceiptPage.tsx

```diff
@@ -129,7 +129,7 @@ export const ReceiptPage = () => {
             <RestartPrompter
               spacingBottom
               restartPath={
-                '/' + ApiDelegationPath.OfferedApiDelegations + '/' + ApiDelegationPath.ChooseOrg
+                '/' + ApiDelegationPath.OfferedApiDelegations + '/' + ApiDelegationPath.ChooseApi
               }
```

Comment:
Not relevant to this specific line, but the 'back' button on the confirmation page needs to go back to the choosOrg page instead of to the chooseApi page as it does now